src/bin/ch-remote.rs

```diff
@@ -516,6 +530,13 @@ fn main() {
                         .help("New memory size (in MiB)")
                         .takes_value(true)
                         .number_of_values(1),
+                )
+                .arg(
+                    Arg::with_name("balloon")
+                        .long("balloon")
+                        .help("New balloon size")
```

Comment:
nit: Mention the units like memory.

Reply:
I changed this part to "help("New memory with balloon size")"